Sources/Prototyper/Login/LoginView.swift

```diff
@@ -46,56 +48,64 @@ struct LoginView: View {
     /// A boolean to check if the user is proceeding without logging in,
     @State var continueWithoutLogin: Bool = false
     
+    var descriptionText: String {
+        if continueWithoutLogin {
+            return LoginViewConstants.DescrpitionText.withoutLoginText
+        } else {
+            return LoginViewConstants.DescrpitionText.withLoginText
+        }
+    }
+    
     
     /// The text to be displayed as placeholder text on the text field.
     var userNamePlaceHolder: String {
         if continueWithoutLogin {
-              return LoginViewConstants.UserNamePlaceHolder.withLoginText
+            return LoginViewConstants.UserNamePlaceHolder.withLoginText
         } else {
-              return LoginViewConstants.UserNamePlaceHolder.withoutLoginText
+            return LoginViewConstants.UserNamePlaceHolder.withoutLoginText
         }
     }
     
     /// The text to be displayed on the Login button.
     var subButtonText: String {
         if continueWithoutLogin {
-              return LoginViewConstants.SubButtonText.withLoginText
+            return LoginViewConstants.SubButtonText.withoutLoginText
         } else {
-              return LoginViewConstants.SubButtonText.withoutLoginText
+            return LoginViewConstants.SubButtonText.withLoginText
         }
     }
     
     var body: some View {
-            VStack {
-                VStack(alignment: .leading) {
-                    Text("Please log in using your Prototyper/TUM credentials to send feedback")
-                        .lineLimit(2)
-                }.padding()
-                    
-                VStack(spacing: 25) {
-                    TextField(userNamePlaceHolder, text: $userid)
-                    if !continueWithoutLogin {
-                        SecureField("Password", text: $password)
-                    }
-                    Button(action: login) {
-                        Text("Login").bold()
+        VStack {
+            VStack(alignment: .leading) {
+                Text(descriptionText)
+                    .lineLimit(2)
+            }.padding()
+            
+            VStack(spacing: 25) {
+                TextField(userNamePlaceHolder, text: $userid)
+                if !continueWithoutLogin {
+                    SecureField("Password", text: $password)
+                }
+                Button(action: login) {
+                    Text("Login").bold()
```

Comment:
Maybe the button text should change based on logging in or just entering a name without a password and logging in.